src/clj/game/cards-resources.clj

```diff
@@ -70,6 +70,18 @@
    {:effect (effect (gain :link 1 :max-hand-size 1))
     :leave-play (effect (lose :link 1 :max-hand-size 1))}
 
+   "Chatterjee University"
+   {:abilities [{:cost [:click 1] :label "Place 1 power counter"
+                 :msg "place 1 power counter on it"
+                 :effect (effect (add-prop card :counter 1))}
+                {:cost [:click 1] :label "Install a program from your Grip"
+                 :prompt "Choose a program to install from your Grip"
+                 :choices {:req #(and (is-type? % "Program") (in-hand? %))}
+                 :msg (msg "install " (:title target))
+                 :effect (effect (install-cost-bonus [:credit (* -1 (:counter card))])
+                                 (runner-install target)
+                                 (add-prop card :counter -1))}]}
```

Comment:
Either this or `add-prop` needs some sort of safeguard to not reduce counters below 0, I was able to install Femme Fatale for 10 with -1 counters on Chatterjee: [image](http://i.share.pho.to/4a9d108d_o.png)